<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineE_tick">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IGLN.L</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tick">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Ticker</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>2</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Nazwa</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_name">
    <property name="geometry">
     <rect>
      <x>128</x>
      <y>14</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>iShares Physical GOLD ETF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Vol">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>1</y>
      <width>54</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Ilość</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_vol">
    <property name="geometry">
     <rect>
      <x>423</x>
      <y>14</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1.3</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_cur">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>13</y>
      <width>30</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>USD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cur">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>-2</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Waluta</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tabela">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>911</width>
      <height>211</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>239</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>160</red>
          <green>239</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>240</red>
          <green>240</green>
          <blue>240</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <column>
     <property name="text">
      <string>Id</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Data</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ticker</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Nazwa</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Waluta</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Kurs</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ilość</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Wartość</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Zmiana</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Zysk</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="b_add">
    <property name="geometry">
     <rect>
      <x>651</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_id">
    <property name="geometry">
     <rect>
      <x>552</x>
      <y>12</y>
      <width>25</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Id">
    <property name="geometry">
     <rect>
      <x>555</x>
      <y>-2</y>
      <width>20</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_update">
    <property name="geometry">
     <rect>
      <x>576</x>
      <y>11</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aktualizuj</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_del">
    <property name="geometry">
     <rect>
      <x>706</x>
      <y>10</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuń</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_wyk">
    <property name="geometry">
     <rect>
      <x>697</x>
      <y>259</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wykres</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_udzial">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>326</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Udziały</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>627</x>
      <y>260</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Kurs</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wart</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Oba</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_kurs">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>441</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>USD/PLN</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_suma">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>259</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>260</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_odsw">
    <property name="geometry">
     <rect>
      <x>153</x>
      <y>326</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Odśwież</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_expo">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>280</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Eksport</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_stats">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>260</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Statystyki</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_curs">
    <property name="geometry">
     <rect>
      <x>499</x>
      <y>12</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>44.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cur_2">
    <property name="geometry">
     <rect>
      <x>503</x>
      <y>-2</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Kurs</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_cash">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>280</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cash">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wolne środki</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_all">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>326</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cash_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Suma aktywów</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_pay">
    <property name="geometry">
     <rect>
      <x>463</x>
      <y>304</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>700</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_limit">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wpłaty na IKZE</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_cost">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>303</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_cash_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>303</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Opłaty</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_data">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>304</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="b_payin">
    <property name="geometry">
     <rect>
      <x>547</x>
      <y>304</y>
      <width>62</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Wpłać</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_limit_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>260</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Suma wpłat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineE_paid">
    <property name="geometry">
     <rect>
      <x>427</x>
      <y>259</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_wynik">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>260</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Wynik</string>
    </property>
   </widget>
   <widget class="QPushButton" name="b_hist">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>283</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Historia</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>284</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>7dni</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1msc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3msc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6msc</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rok</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="b_sell">
    <property name="geometry">
     <rect>
      <x>754</x>
      <y>9</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sprzedaj</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
